lib/resources/majors.tsx

```diff
@@ -4365,6 +4356,382 @@ const minorAMS_New: Minor = {
   ],
 };
 
+const bsNeuro: Major = {
+  degree_name: 'B.S. Neuroscience',
+  abbrev: 'B.S. Neuro',
+  department: 'AS Neuroscience',
+  total_degree_credit: 120,
```

Comment:
I forget what we spoke about during the meeting when discussing this, but total_degree_credit should be the direct requirements of the major